src/plugins/janus_sip.c

```diff
@@ -1047,6 +1050,27 @@ typedef struct janus_sip_media {
 	int audio_level_extension_id;
 } janus_sip_media;
 
+typedef struct janus_rtp_rfc2833_payload {
+#if __BYTE_ORDER == __BIG_ENDIAN
+	uint8_t event;
+	uint8_t end : 1;
+	uint8_t reserved : 1;
+	uint8_t volume : 6;
+	uint16_t duration;
+#elif __BYTE_ORDER == __LITTLE_ENDIAN
+	uint8_t event;
+	uint8_t volume : 6;
+	uint8_t reserved : 1;
+	uint8_t end : 1;
+	uint16_t duration;
+#endif
+} janus_rtp_rfc2833_payload;
```

Comment:
I think this struct belongs to `rtp.h`, where we define other RTP and RTCP related properties. This will allow other plugins interested in using that payload to refer to an existing structure.

As a side note, for code style don't put any spaces in the size specifier (so it should be, e.g., `volume:6` and not `volume : 6`).

Reply:
I moved this struct definition to rtp.h in my last commit.